test/features/support/voucher_cancelation.js

```diff
@@ -0,0 +1,129 @@
+const pactum = require('pactum');
+const { Given, When, Then, Before, After } = require('@cucumber/cucumber');
+const { localhost } = require('./helpers/helpers');
+
+let specVoucherCancelation;
+let voucherSerialNumber;
+let voucherCancelationBodyBody;
+
+const baseUrl = `${localhost}vouchers/voucherstatuscheck/{voucherserialnumber}`;
+const requestFunction = () =>
+  specVoucherCancelation.patch(baseUrl).withBody(voucherCancelationBodyBody)
+  .withPathParams('voucherserialnumber', voucherSerialNumber);
+
+Before(() => {
+  specVoucherCancelation = pactum.spec();
+});
+
+// Scenario: The user successfully cancels a voucher
+Given('The user wants to cancel a voucher', () => {
+  voucherSerialNumber = '1234';
+  voucherCancelationBodyBody = {
+    voucherserialnumber: voucherSerialNumber,
+    Gov_Stack_BB: 'Gov_Stack_BB'
+  };
```

Comment:
You are writing the same code three times. This part can be in constants, for example `validVoucherCancelationBody` and then 
`voucherCancelationBodyBody = validVoucherCancelationBody`

Reply:
good idea, will change it quick